provision/filter_plugins/aws_sg_rules.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
My copypasta there was more to describe the ruleset that I think we should have rather than the formatting.  I copied that from some old code I had laying around as an example.  Sorry for the confusion - I just think that we should have the following rules by default:

*inbound ping allowed
*inbound SSH allowed
*outbound access allowed